bin/vagrant.sh

```diff
@@ -12,3 +12,25 @@ mkdir -pm 700 /home/vagrant/.ssh
 curl -L https://raw.githubusercontent.com/mitchellh/vagrant/master/keys/vagrant.pub -o /home/vagrant/.ssh/authorized_keys
 chmod 0600 /home/vagrant/.ssh/authorized_keys
 chown -R vagrant:vagrant /home/vagrant/.ssh
+
+echo ">>> Modifying sshd_config to speedup login"
+function edit_sshd_config
```

Comment:
I'm not excited about using a function here. Can you refactor it into a for loop? Maybe use an associative array. There's also some styling things that need cleanup.